### ISSUE TITLE
Fiscal hosting as a service landing page

### ISSUE THREAD

**Issue by Memo-Es** [opencollective/opencollective#4118]
## Background
When onboarding new projects, hosts usually need to send people interested resources and documentation explaining what a fiscal host is and how fiscal hosting can help those projects. This is usually hard because they have to spend time looking for the right references or having calls to explain all about fiscal hosting.

We want hosts to have a branded tool they can send to their prospects, telling all that's necessary to know about fiscal hosting and how it can help several types of projects succeed, using a simple and friendly language that conveys trust.

## Design and copywriting

![Desktop](https://user-images.githubusercontent.com/49539954/117332725-c679e180-ae5d-11eb-9df6-0034aadb5b73.png)

[Link to the copywriting document](https://docs.google.com/document/d/1-6OazAPai01A0dJvMk6Dx-LgwPYc1B-Y6yM_TTuM63w/edit?usp=sharing)

Feel free to leave feedback or thoughts in the comments!

**Comment by SudharakaP** [comment]
Small suggestion I was thinking; this page gives the idea that one has to join an existing fiscal host. There's a link near the end of the page `Become a fiscal host` and I thought it can become a bit confusing to a newcomer as all the content above it doesn't mention about becoming your own fiscal host; the language implies that one has to join an existing fiscal host. I would suggest adding that link somewhere on the top (maybe at the end of `What is a fiscal host`? 🤔) and/or adding some content to the page so that people would know they can also become their own host. 😄 

Basically my idea is that if there's a link at the bottom that says `Become a fiscal host` the content above should explain or support it. Alternatively we could also remove that `Become a fiscal host` link if the only purpose of the page is to onboard new project by existing hosts. 😄

**Comment by patcon** [comment]
This looks rad! I feel that this would be useful, but also, there's maybe some friction about the idea of trying to grow a host to sustainability, but then sending prospects to a page that might lose people to other bigger hosts. Would it be possible to use a querystring in the url to create a bigger call-to-action for our own host?

Also, it might be nice to use query strings to "hide" certain sections that don't fit the motivations or audience. Specifically, I'm looking at the "find a fiscal host that's right for you", which won't make sense if the CTA is for registering for one specific host :)

Any minor nit: In the "time-limited project" section, "set _up_" then "wind _down_" lingo feels nicer:

> where it doesn't make sense to set up a whole new organization only to wind it ~~up~~ _down_ six months later

**Comment by alanna** [comment]
Reposting my slack comment: Looks great! Seems the content for Apply To One of Our Fiscal Hosts is not done? Since our host discovery design is not great, could we actually list six hosts here instead of three? I suggest OSC, OCF, SCA, AllForClimate, OC Europe… and one more?

**Comment by alanna** [comment]
Sure if you want OCNZ... but maybe we should more objectively just go by host size or activity to make it fair.

**Comment by alanna** [comment]
Can closing issues please include things like the URL to the finished page? How do I access this? Has someone demoed it in the demos channel? Is it documented?

**Comment by snitin315** [comment]
@alanna here - https://opencollective.com/fiscal-hosting

**Comment by alanna** [comment]
Thank you! It looks great. Is it linked from anywhere?

**Comment by snitin315** [comment]
@alanna yes we have a link in the footer. `About fiscal hosting` under the `join` column.

**Comment by alanna** [comment]
Thanks! I've shared this ins #hosts and #newsletter on Slack. Would be good to hear some input from the team about how we can get this out there and make sure people know it's there as a resource.

**Comment by SudharakaP** [comment]
> Thanks! I've shared this ins #hosts and #newsletter on Slack. Would be good to hear some input from the team about how we can get this out there and make sure people know it's there as a resource.

Two suggestions I have are,

- Adding the fiscal hosts page to https://docs.opencollective.com/help/fiscal-hosts/become-a-fiscal-host
- Adding the fiscal host page link to the `Become a Fiscal Hosts` section; 

![image](https://user-images.githubusercontent.com/12435965/119759003-4e9b4780-be5c-11eb-967b-eb06075fce04.png)

Currently the page seems very hidden and hard to find for a newcomer. 🐼

**Comment by znarf** [comment]
> Can closing issues please include things like the URL to the finished page? How do I access this? Has someone demoed it in the demos channel? Is it documented?

@alanna To give some context, the closing was automated when I merged the related PR, not a manual action.

**Comment by alanna** [comment]
Feedback: 

> The fiscal host page looks great! thanks for that! could I ask a little favour: the logo of A4C has changed, is it possible to change that on the OC website, when showing different fiscal hosts? And something else: although we love Extinction Rebellion, it scares people off and we need to grow beyond movements like XR to become sustainable. It would be great if the description of A4C would be: host (or shared back office) dedicated to support movements working on climate and social justice.

**Comment by Memo-Es** [comment]
Closing it since it's online at [https://opencollective.com/fiscal-hosting](https://opencollective.com/fiscal-hosting)
